src/python/pants/backend/python/target_types.py

```diff
@@ -858,6 +858,25 @@ class PythonTestsExtraEnvVarsField(TestExtraEnvVarsField):
     pass
 
 
+class PythonTestsXdistConcurrencyField(IntField):
+    alias = "xdist_concurrency"
+    help = softwrap(
+        """
+        Number of CPUs that should be used to run each test file belonging to this target.
```

Comment:
`{pants_concurrency}` is a bit confusing at first, but the jist is we can request a _maximum_ amount of CPUs we'll need from the engine, and then the engine gives us that or fewer (either fewer CPUs are available, or they ar ebeing consumed by other processes). In this case, I wouldn't want the user getting 1 CPU when they set 4 and wondering what's wrong.

```suggestion
        Maximum number of CPUs to allocate to run each test file belonging to this target.
```